css/home.css

```diff
@@ -7,7 +7,7 @@ body {
   cursor: default;
 }
 .navbar {
-  margin: 4% 6% 5% 7%;
+  margin: 2% 6% 5% 7%;
```

Comment:
idk if it's just on my machine but this is how it looks for me

![Screenshot from 2020-12-29 14-37-17](https://user-images.githubusercontent.com/63963181/103272633-67095200-49e3-11eb-8530-e1707920492c.png)

in my opinion it'd look better if the whole navbar is on top